app/api/user.py

```diff
@@ -0,0 +1,35 @@
+from fastapi import APIRouter, status, Response
+from app.models.models import User
+from app.service.user import read_users, read_user, create_user, update_user, delete_user
+
+router = APIRouter()
+
+
+@router.get('/users', status_code=status.HTTP_200_OK, response_model=list[User])
+async def get_all_users():
+    results = await read_users()
+    return results
+
+
+@router.get('/users/{user_id}', status_code=status.HTTP_200_OK)
+async def get_user(user_id: int):
+    results = await read_user(user_id)
+    return results
+
+
+@router.post('/users', status_code=status.HTTP_201_CREATED)
+async def create_a_user(name: str, email: str, password: str):
```

Comment:
```suggestion
async def create_user(name: str, email: str, password: str):
```
관사는 함수나 변수 명명시 필수적으로 넣고 있지는 않아서 빼도 좋을 것 같아요.
다른 부분에도 같이 적용 부탁드릴께요.

이 부분만 수정하시고 머지해주시면 될 것 같아요

Reply:
수정 완료했습니다~!